src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/resources/doclint.properties

```diff
@@ -27,12 +27,12 @@ dc.anchor.already.defined = anchor already defined: "{0}"
 dc.anchor.value.missing = no value given for anchor
 dc.attr.lacks.value = attribute lacks value
 dc.attr.not.number = attribute value is not a number
-dc.attr.not.supported.html4 = attribute not supported in HTML4: {0}
 dc.attr.not.supported.html5 = attribute not supported in HTML5: {0}
 dc.attr.obsolete = attribute obsolete: {0}
 dc.attr.obsolete.use.css = attribute obsolete, use CSS instead: {0}
```

Comment:
Is this still required?

Reply:
Do you mean ".html5" would be better to be removed?

Reply:
Sorry, the comment is incorrectly focussed (and annoyingly cryptic: my apologies). 
I think I was referring to `dc.attr.obsolete.use.css`

Reply:
I got it.  Ok. Will remove it.